src/main/java/org/apache/commons/io/FileUtils.java

```diff
@@ -2698,7 +2691,7 @@ private static void requireAbsent(final File file, final String name) throws Fil
      *
      * @param file1 The first file to compare.
      * @param file2 The second file to compare.
-     * @throws IOException if an I/O error occurs.
+     * @throws IOException              if an I/O error occurs.
```

Comment:
Ups... auto-formatting. I'll revert the whitespace changes from methods which are not part of IO-810.